### DISCUSSION TITLE
Welcome to SecureDNSClient Discussions!

### DISCUSSION THREAD

**Discussion by msasanmh** [msasanmh/DNSveil#186] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below and tell us about what you do with `SDC - Secure DNS Client`.

**Comment by deathline94** [comment]
Huge respect, bro. SDC was quite handy in more than dozens occasions for me.
Tiny bit of request tho. Add something like “import servers from TXT file” and make the process of installing
the cert a bit easier and more streamlined. Also, dragging the SDC window around is a bit wonky (don't know if its on purpose or not).
Again, hats off !!

**Reply by msasanmh** [reply]
I'm glad you found it useful.

You already can import servers from .txt files:
Custom Servers  -> New Group  -> Add Servers  ->  Add Multiple (Browse).

About the certificat,  I'll see what I can do.

And about dragging the window... In the future,  I'm going to write SDC Plus and then I can use a newer GUI technology, but it will only support 64 bit systems and Windows 10 above.

**Reply by deathline94** [reply]
Hat's off, sir ! 🤘
